src/state/apis/swappers/swappersApi.ts

```diff
@@ -74,12 +74,12 @@ export const swappersApi = createApi({
         const orderedQuotes: ApiQuote[] = orderBy(
           quotesWithInputOutputRatios,
           ['inputOutputRatio', 'swapperName'],
-          ['asc', 'asc'],
+          ['desc', 'asc'],
         )
 
-        dispatch(tradeQuoteSlice.actions.setQuotes({ quotes: orderedQuotes, queryStartTime }))
+        dispatch(tradeQuoteSlice.actions.setSwapperName(orderedQuotes[0]?.swapperName))
```

Comment:
We still need to decide how we handle swapper selection resets on new quotes. Leaving as-is behaviour for now.